news/2022-10-21-project-loved-october-2022.md

```diff
@@ -0,0 +1,374 @@
+
+So let me see, I will just put it up for loved like this... and now people can vote for it! But, before you do, let me take you for a spin! Here, let me just turn on score V2 and we're off to- to... to the... what was I saying agai- *yawn* ......................................
+
+## osu!mania
+
+The osu!mania Loved candidates were chosen by [\_underjoy](https://osu.ppy.sh/users/2235750), [-NoName-](https://osu.ppy.sh/users/10400730), [kadoen](https://osu.ppy.sh/users/12780575), [Paturages](https://osu.ppy.sh/users/1375479), [RandomeLoL](https://osu.ppy.sh/users/7080063) and [stupud man](https://osu.ppy.sh/users/2141612)!
+
+---
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/1569250.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[sabi - true DJ MAG top ranker's song Zenpen (katagiri Remix)](https://osu.ppy.sh/beatmapsets/1569250#mania) by [Ciel](https://osu.ppy.sh/users/1455309)\
+170 BPM, 4:22 | 4.19★, 4.19★, 4.46★, 4.46★\
+*description written by [Paturages](https://osu.ppy.sh/users/1375479)*
+
+Remember [*lost found melody*](https://osu.ppy.sh/beatmapsets/1051052)? [*Shakunetsu*](https://osu.ppy.sh/beatmapsets/853414)? Well, [Ciel](https://osu.ppy.sh/users/1455309) is back for more with *true DJ MAG*!
+
+As someone who has been involved with SV pooling for MWC 2021, I deeply recall the crunch week we spent on trying to deliver this map for the Finals mappool. Yes, this was indeed supposed to be the memorization-oriented SV entry for MWC 2021, but it was just way too hard and unfair for that competitive setting in particular!
+
+As such, you can find 2 sets of difficulties in this mapset: *top=false* features nerfed rice patterns that have been iterated during MWC 2021 pooling in order to try and make the pick more approachable compared to [*Mafain*](https://osu.ppy.sh/beatmapsets/1569493), which was the other SV pick for that stage; *top=true* features the unnerfed version, true to Ciel's vision and direction for this map. Both versions however feature the same, unforgiving, brutal SV gimmicks that one has come to expect from Ciel over the years.
+
+The barrier of entry to even try and grasp this map is incredibly high: this challenge and this leader board is reserved only for the cream of the crop, the best of the best of this community. If you want to make a name for yourself and think you're capable of catching a spot among the best SV players in the world, this might be the map you're looking for. Good luck on learning, have fun grinding.
```

Comment:
```suggestion
The barrier of entry to even try and grasp this map is incredibly high: this challenge and this leaderboard is reserved only for the cream of the crop, the best of the best of this community. If you want to make a name for yourself and think you're capable of catching a spot among the best SV players in the world, this might be the map you're looking for. Good luck learning, have fun grinding.
```